_plugins/sidebar.rb

```diff
@@ -68,20 +68,36 @@ def sanitize_filename(filename)
       sanitized_filename.gsub('.html', '')
     end
 
-    def generateSubgroup(filename, _key, value, has_subgroups)
+    def generateSubgroup(filename, key, value, all_subgroups, level)
```

Comment:
Codacy found an issue: [Cyclomatic complexity for generateSubgroup is too high. [12/6]](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5781626)